origin-growth/src/apollo/resolvers.js

```diff
@@ -76,11 +77,13 @@ const resolvers = {
     }
   },
   Mutation: {
-    async invite() {
+    async invite(root, args) {
+      // FIXME: Check the referrer against Auth token.
+      await sendInviteEmails(args.walletAddress, args.emails)
       return {
-        code: '418',
-        success: false,
-        message: 'I am a teapot'
+        code: '200',
```

Comment:
@sparrowDom I was not sure what values to return here ? What is the intent of code, success and message exactly ?

Reply:
I think we can remove the `code` since that should be communicated with the `http.code`. The message I thought would serve as an error reporting field

Reply:
Sounds good. Let me remove it.
I actually wonder if we should just rely on standard apollo responses for now and not bother about success and message neither ? My reasoning:
 - I don't think we are returning any user friendly messages from the back-end at this point ? Or do you have any specific use cases in mind ?
 - messages would need to get translated which would take some extra work/thinking
Perhaps we could add this later ? Let me know, I don't really feel strongly about this, just trying to keep things simple... :)

Reply:
Yeah good point, agree with keeping it simple for now. And add complexity in case it is needed later.

Reply:
Ok. Let me take this out for now...